tsconfig.eslint.json

```diff
@@ -8,5 +8,6 @@
   "include": [
     "eslint.config.js",
     "rollup.config.js",
+    "*.ts",
```

Comment:
why this is needed?

Reply:
To cover all possible typescript files under all projects. This won't enable eslint to do lint on all typescript files since we run eslint with paths to lint.

Reply:
doesn't it work like that already?

Reply:
Instead, we can specify everything in `include` to each of `tsconfig.json`.

Reply:
Both `tsconfig` and `eslint.config` should include the file to enable linting.

Reply:
I would do that :)